src/components/common/Button/index.tsx

```diff
@@ -3,14 +3,15 @@ import React from 'react';
 interface Props {
   text: string;
   type: string;
+  maxWidth?: string;
   variant: 'primary' | 'secondary' | 'subtle';
   onClick?: React.MouseEventHandler<HTMLButtonElement>
 }
 function Button({
-  text, type = 'button', variant = 'primary', onClick,
+  text, type = 'button', variant = 'primary', onClick, maxWidth = 'w-full',
 }: Props) {
   return (
-    <button onClick={onClick} type={type} className={`btn btn-${variant}`}>{text}</button>
+    <button onClick={onClick} type={type} className={`btn btn-${variant} ${maxWidth}`}>{text}</button>
```

Comment:
For dynamic styling like this we usually use the library `classnames`, string interpolation for more complex cases can get a little hard to implement/ hard to read.
Using classnames this could be something like:
```
import cn from 'classnames'
.....

<button ... className={cn('btn', `btn-${variant}`, maxWidth)}>...</button>
```

Reply:
Perfect! Seems much more readable